source.ipynb

```diff
@@ -38,7 +48,16 @@
    "source": [
     "## What would an answer look like?\n",
     "*What is your hypothesized answer to your question?*\n",
-    "📝 <!-- Answer Below -->"
+    "\n",
+    "1. Geographical barriers can be a barrier, such as a patient living in a rural area, this barrier can be addressed with telehealth. Therefore, hospitals that lack specialists will see improved health outcomes or diagnosis with the assistance of telehealth. \n",
+    "\n",
+    "2. Socioeconomical barriers can be addressed with communication, educational resources, and addressing/listening to the patient's health literacy.\n",
+    "\n",
+    "3. The patient's health literacy is strongly impacting the access to healthcare. Health literacy is taught, and the patient might not understand when or why they should go in for care. Or, they might not understand why the treatment plan is setup the way it is. \n",
+    "\n",
+    "5. With there being so many different cultures, the lack of cultural competence will have a big impact on the access to care for minority groups.\n",
+    "\n",
+    "6. The beliefs for Eastern medicine is going to have a big impact on a patient following their treatment plan that follows Western medicine. "
```

Comment:
I can see you have done a lot of research into what is causing the barriers. You have some interesting ideas about how some of those issues can be resolved. I'm not understanding how you plan to demonstrate these with data, however. 

Geographical barriers could be demonstrated with a heat map that shows how healthcare facilities are concentrated in comparison to population density. Alternatively, you could use a histogram where the x-axis has bins of population ranges and the y-axis represents the number of healthcare facilities per population range.

For your other "what would an answer look like" topics, I am not certain how you would represent those or even if you can. The version of this file that I am reviewing doesn't have specific data sources or any mention of how you would combine them. I don't know how you would be able to discover that patients believe in Eastern medicine and that patients who believe in Eastern medicine do not follow their treatment plans. Due to the anonymization of healthcare data, you may have a hard time linking that belief and the outcome.

You have offered some solutions like "communication, educational resources, and addressing/listening to the patient's health literacy," but I don't understand how you would show a direct link between those solutions and patient outcomes.

You may need to limit your scope to what can be directly demonstrated (limited access to healthcare centers based on location) or other quantitative data.